ballerina/queue_manager.bal

```diff
@@ -22,7 +22,7 @@ public isolated class QueueManager {
         check self.externInit(configurations);
     }
 
-    isolated function externInit(QueueManagerConfiguration configurations) returns Error? = @java:Method {
+    private isolated function externInit(QueueManagerConfiguration configurations) returns Error? = @java:Method {
```

Comment:
```suggestion
    isolated function externInit(QueueManagerConfiguration configurations) returns Error? = @java:Method {
```

Reply:
we usually make the extern function private right? Otherwise these are suggested in places like tests

Reply:
Removed